src/igvfd/schemas/mixins.json

```diff
@@ -266,5 +266,29 @@
                 "linkTo": "Document"
             }
         }
+    },
+    "product_id": {
```

Comment:
This does not need to be done on this commit, but I think we will thank ourselves later for making product_id, source and lot_id a set the way that "submitted" is a set.

Reply:
Agreed, better to handle with source